packages/viewer/package.json

```diff
@@ -45,12 +45,13 @@
   },
   "dependencies": {
     "@types/classnames": "2.2.3",
-    "@vertexvis/frame-streaming-protos": "^0.1.4",
+    "@vertexvis/frame-streaming-protos": "^0.1.5",
     "@vertexvis/geometry": "0.9.0",
     "@vertexvis/stream-api": "0.9.0",
     "@vertexvis/utils": "0.9.0",
     "classnames": "2.2.6",
     "protobufjs": "^6.9.0",
+    "resize-observer-polyfill": "^1.5.1",
```

Comment:
types coming to typescript soon: https://github.com/microsoft/TSJS-lib-generator/issues/908

wide support in the latest versions of modern browsers, but nothing in IE